fix(serverless-agent) Add ignore containers support to TF data source (#369)
sysdiglabs/terraform-provider-sysdig: website/docs/d/fargate_workload_agent.md

  * `stream_prefix` - Prefix for the instrumentation log stream
  * `region` - The AWS region where the target log group resides
* `sysdig_logging` - (Optional) The instrumentation logging level: `trace`, `debug`, `info`, `warning`, `error`, `silent`.

> * `ignore_containers` - (Optional) A list of containers in this data source that should not be instrumented.

Reviewer: ```suggestion
* `ignore_containers` - (Optional) A list of containers in the task definition that should not be instrumented.
```
